src/main/resources/_config.properties

```diff
@@ -39,13 +39,14 @@ init_retry_interval=1
 #=====================================================#
 #uncomment and change udid and deviceName for Android test
 #============ Android Local Mobile ===================#
-#selenium_url=http://localhost:4723/wd/hub
-#capabilities.automationName=uiautomator2
-#capabilities.deviceName=<phone_name>
-#capabilities.udid=<phone_udid>
-#capabilities.deviceType=phone
-#capabilities.platformName=ANDROID
-#capabilities.app=https://qaprosoft.s3-us-west-2.amazonaws.com/carinademoexample.apk
+selenium_url=http://localhost:4723/wd/hub
+capabilities.automationName=uiautomator2
+capabilities.deviceName=Pixel 6 API 33
+capabilities.udid=emulator-5554
+capabilities.deviceType=phone
+capabilities.platformName=ANDROID
+capabilities.app=/Users/akhivyk/Downloads/com.android.chrome.apk
```

Comment:
you dont need to attach chrome apk to test webside, as chrome is natively installed on Android